src/plugins/ramps/paybis/paybisRampPlugin.ts

```diff
@@ -0,0 +1,1360 @@
+          method: 'GET',
+          url,
+          path: `v2/public/currency/pairs/sell-crypto`,
+          apiKey
+        })
+        paybisPairs.sell = asPaybisSellPairs(response)
+      } catch (e) {
+        console.error(String(e))
+      }
+    }
+
+    if (paybisPairs.sell != null) {
+      for (const paymentMethodPairs of paybisPairs.sell.data) {
+        const { name, pairs } = paymentMethodPairs
+        if (name == null) continue
+        const edgePaymentType = PAYMENT_METHOD_MAP[name]
+        if (edgePaymentType == null) continue
+        for (const pair of pairs) {
+          const { fromAssetId, to } = pair
+
+          let paymentMethodObj = allowedCurrencyCodes.sell[edgePaymentType]
+          if (paymentMethodObj == null) {
+            paymentMethodObj = { providerId: pluginId, crypto: {}, fiat: {} }
+            allowedCurrencyCodes.sell[edgePaymentType] = paymentMethodObj
+          }
+
+          const edgeTokenId = PAYBIS_TO_EDGE_CURRENCY_MAP[fromAssetId]
+          if (edgeTokenId == null) continue
+          const { pluginId: currencyPluginId } = edgeTokenId
+
+          let tokens = paymentMethodObj.crypto[currencyPluginId]
+          if (tokens == null) {
+            tokens = []
+            paymentMethodObj.crypto[currencyPluginId] = tokens
+          }
+          addTokenToArray({ tokenId: edgeTokenId.tokenId }, tokens)
+
+          for (const fiat of to) {
+            paymentMethodObj.fiat[`iso:${fiat}`] = true
+          }
+        }
+      }
+    }
+  }
+
+  const ensureStateInitialized = async (): Promise<void> => {
+    if (state == null) {
+      const { apiKey, partnerUrl, privateKeyB64 } = initOptions
+
+      let partnerUserId: string
+      if (pluginConfig.store != null) {
+        partnerUserId = await pluginConfig.store
+          .getItem('partnerUserId')
+          .catch(() => '')
+        if (partnerUserId === '' && pluginConfig.makeUuid != null) {
+          partnerUserId = await pluginConfig.makeUuid()
+          await pluginConfig.store.setItem('partnerUserId', partnerUserId)
+        } else if (partnerUserId === '') {
+          partnerUserId = `edge-user-${Date.now()}-${Math.random()
+            .toString(36)
+            .substring(7)}`
+          await pluginConfig.store.setItem('partnerUserId', partnerUserId)
+        }
+      } else {
+        partnerUserId = `edge-user-${Date.now()}-${Math.random()
+          .toString(36)
+          .substring(7)}`
+      }
+
+      state = {
+        apiKey,
+        partnerUrl,
+        privateKeyB64,
+        partnerUserId
+      }
+    }
+  }
+
+  const ensureAssetsInitialized = async (
+    direction: 'buy' | 'sell'
+  ): Promise<void> => {
+    await ensureStateInitialized()
+
+    if (direction === 'buy') {
+      await initializeBuyPairs()
+    } else {
+      await initializeSellPairs()
+    }
+  }
+
+  const validateSupportRequest = (
+    regionCode?: FiatPluginRegionCode,
+    countryCode?: string
```

Comment:
country code is in the regionCode so just pass region

Reply:
Remove `countryCode` parameter from `validateSupportRequest` signature and instead use `regionCode.countryCode` in it's place.